### PR TITLE
Add WNT API examples

### PR DESCRIPTION
**Pull request by teemupiiroinenwirepas**
Signed-off-by: Teemu Piiroinen <teemu.piiroinen@wirepas.com>

### CODE REVIEW

File: wrappers/python/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # grpc
 grpcio==1.23.0
 protobuf==3.9.2
+websocket-client==0.48.0
```
Comment (PFigs) [inline wrappers/python/requirements.txt]: Please remove it, as it is not needed for the wirepas-messaging packet. 

lets add a requirements inside the examples folder instead

Comment (teemupiiroinenwirepas) [reply wrappers/python/requirements.txt]: Do I need to add anything else than the requirements file? Packet creator can find it automatically?

Comment (PFigs) [reply wrappers/python/requirements.txt]: Drop a requirements.txt inside the examples/wnt with whatever extra package you need

Comment (teemupiiroinenwirepas) [reply wrappers/python/requirements.txt]: Can you check the latest code. I hope that it is fixed there.

**Pull request merged**